contracts/Forwarder.sol

```diff
@@ -136,6 +136,7 @@ contract Forwarder is IERC721Receiver, ERC1155Receiver, IForwarder {
     uint256 value,
     bytes calldata data
   ) external onlyParent returns (bytes memory) {
+    require(target != address(0), 'Invalid target address');
```

Comment:
This method and some other methods were highlighted so as to add zero address validation before assignment or making external calls, potentially to avoid the use of zero addresses which may lead to unexpected behavior or vulnerabilities.

Reply:
Removed all zero address validations to save on gas